### PR TITLE
feat(Podurama): add presence

### PR DESCRIPTION
**Pull request by ION606**
## Description
resolves https://github.com/PreMiD/Presences/issues/7428

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>

image

image

image

image

<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/P/Podurama/metadata.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Podurama",
+	"description": {
+		"en": "Google Podcast presence, but with the podcast artwork"
+	},
+	"url": "podurama.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/CNkpXlB.png",
```
Comment (theusaf) [inline websites/P/Podurama/metadata.json:13]: [Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) should be 512x512px

File: websites/P/Podurama/metadata.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	"thumbnail": "https://i.imgur.com/CNkpXlB.png",
```
Comment (theusaf) [inline websites/P/Podurama/metadata.json:14]: The [thumbnail](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#thumbnail) should preferably be a [wide promotional card](https://i.imgur.com/3QfIc5v.jpg) or a [screenshot](https://i.imgur.com/OAcBmwW.png) if the first is not available.

File: websites/P/Podurama/presence.ts [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		[, sect, subsect] = document.location.pathname.split("/"),
+		player = document.querySelector(".audio-player"),
+		usetimeleft = await presence.getSetting<boolean>("usetimeleft");
```
Comment (theusaf) [inline websites/P/Podurama/presence.ts:33]:
```suggestion
		useTimeLeft = await presence.getSetting<boolean>("usetimeleft");
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/P/Podurama/metadata.json (hunk 1 above)
Comment (ION606) [reply websites/P/Podurama/metadata.json:13]: oh. I took the logo from the issue so I assumed it was good my bad

File: websites/P/Podurama/metadata.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Podurama",
+	"description": {
+		"en": "Google Podcast presence, but with the podcast artwork"
```
Comment (theusaf) [inline websites/P/Podurama/metadata.json:9]: The official description may be better
```suggestion
		"en": "Best free podcast player to listen to free podcasts, add custom rss feeds or upload your private files on mobile and desktop devices. Discover over 2M podcasts and millions of episodes."
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**